### DISCUSSION TITLE
a value in the starter Tuning file for NAFs with A4 fundamental

### DISCUSSION THREAD

**Discussion by JeffBurris** [edwardkort/WWIDesigner#116] [NAF Study] [answered]
Hello,
As always, thanks to Edward Kort and his collaborators.

Please excuse me if this has been covered elsewhere but I noticed in that, in regard to the NAF starter Tuning file for flutes with a fundamental of A4 (in v. 2.6.0 and I believe the one just prior) the default target value for G5 is 782.6300 Hz instead of the value of 783.99 Hz I see in reference materials elsewhere. 
The changing of this does not make any difference to my flute, as I'm always expected (and do arrive) a smidge over 784 anyway, give or take; it merely, slightly alter the deviation report inconsequentially.  

I was just wondering if it was an error I should correct or if there was a reason there was a sort of manual offset being applied to this pitch. I also have not confirmed I haven't somehow, inadvertently corrupted the original file, myself.

Thank you,
Jeff

**Comment by edwardkort** [comment, marked as answer]
Hi Jeff,

That is an interesting observation! When I go back through my old tuning files, I find similar discrepancies in the intervals across many keys. It's like I generated the tuning files using an incorrect temperament file. Who knows, it was a lot of years ago. But the sample tuning files are likely all incorrect.

When I recreate an A4 equal-temperament scale in the Tuning Wizard, the intervals are correct - and G5 is correct at 783.99 Hz.

I suggest that you create your tuning files de novo, using the Tuning Wizard. And I will replace the sample tuning files in the next release.

Thank you for the good catch.

Cheers,
Edward Kort
Wood Wind

**Reply by JeffBurris** [reply]
Thanks very much. I'll make them up freshly moving forward. I'm going to try a G4 now and work down (and even up). Very exciting and rewarding.

The question is definitely answered; forgive me if there is a way for me to mark it answered but it's looking to me like it requires moderator status, as I cannot locate a way to do so.

Thanks again Mr. Kort!
You and most of the people I've run into in the NAF spheres are so generously helpful. I will return the favor to whomever I can as I gain experience.

**Reply by edwardkort** [reply]
Jeff,

I marked this as answered. There is supposed to be a way for the author to mark a question as answered, but this is still a new feature in GitHub. In actuality, this is really an issue, to be tracked in our code commits. But again, I could not see a way of easily moving it to Issues. I will open a new Issue so I don't forget.

Although many of us sell our flutes, we don't look on this as a competition. Most freely share their experience and expertise. One of the reasons I really miss flute festivals: the get togethers with other flute makers to share.

Cheers,
Edward
